### ISSUE TITLE
Delta Deployment task tries to copy renamed/deleted file

### ISSUE THREAD

**Issue by florianhoehn** [amguerrero/sfdc_ant_tasks#6]
I am running into the issue that the delta deployment task tries to copy a renamed file which does not exist anymore.

Log:
`BUILD FAILED
java.nio.file.NoSuchFileException: /path/<file that got renamed>`

**Comment by florianhoehn** [comment]
This also occurs if a file got deleted after the tag.

**Comment by amguerrero** [comment]
Hi Florian,

I think I have fixed this issue (and the #7 ), now if a file doesn't exist anymore instead of trying to add it to the delta package it adds it to a destructive changes package.
By default **deltaDeployment** task creates the destructive changes in the *destructiveChanges/* directory, but it can be changed by adding the **destructiveFolder** attribute, like this:
```
<deltaDeployment
    deltaFolder="delta"
    destructiveFolder="anotherDestructiveChanges" ... />
```

And now deltaFolder is optional as well, if you don't add it, it will use the *delta/* for the delta directory.

Can you give it a try and let me know if it's all working properly for you?
